cli/data/playbooks/setup/setup.yaml

```diff
@@ -14,7 +16,6 @@
       ansible.builtin.package:
         name:
           - podman
-          - python3-cryptography
           - python3-libsemanage
           - python3-requests
           - bash-completion
```

Comment:
2 lower is also `python3-psycopg2` which needs to be moved into `setup.py` as a proper runtime dependency because `community.postgresql` depends on it